### PR TITLE
feat(dashboard): replace highlight selection with arrow indicator

### LINKED ISSUE #144: Dashboard: Replace highlight selection with arrow indicator
Replace the current highlight-based selection indicator (inverse text) with a visual arrow or pointer indicator, like other CLI tools use.

**Current behavior:**
- Selected row uses inverse video (`bold={selected} inverse={selected}`)
- Makes the entire row highlighted

**Desired behavior:**
- Add an arrow or pointer (e.g., `→` or `▶`) to the left of the selected row
- Remove or minimize the highlight styling
- Match conventions used by other CLI tools (fzf, ripgrep, etc.)

**Implementation notes:**
- Update `DispatchTable.jsx` to render indicator in first column or prefix
- Keep the selection logic unchanged
- Ensure indicator is visible and clear

**Parent:** #142

### PR DESCRIPTION
**Pull request by jsturtevant**
## Summary

Replace the full-row inverse video selection indicator in the dashboard with a leading `❯` arrow, matching CLI conventions (fzf, inquirer).

## Changes

- **DispatchTable.jsx**: Added a 2-char leading column with cyan `❯` indicator for the selected row, space for unselected. Removed `inverse={selected}` styling, kept `bold={selected}`. Added matching spacer in the header row.
- **DispatchTable.test.js**: Updated selection tests to assert on the arrow indicator instead of inverse styling.

## Before / After

| Before | After |
|--------|-------|
| Full inverse-highlighted row | `❯` arrow + bold on selected row |

All tests pass (34/34).

Closes #144

### CODE REVIEW

File: .squad/log/2026-02-23T2155-read-only-dispatch.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Copilot Review
+
+5 comments addressed:
+- Path validation improvements
+- Test naming conventions
+- Enhanced test coverage for edge cases
+
+## Outcome
+
+PR #141 merged to main. Feature enforces read-only mode as designed. All tests passing.
```
[the remark this answers is not in the document]
Comment (jsturtevant) [reply .squad/log/2026-02-23T2155-read-only-dispatch.md]: Same as above — squad session logs are committed on the active branch by the Scribe. This is working as designed with the worktree-local strategy.
